docs/blog/20230801-Daeraxa-AugustUpdate.md

```diff
@@ -0,0 +1,92 @@
+
+# Welcome to the August Community Update
+
+As always, a big, warm welcome to this month's update! This time around we have some interesting new features, some of which have been in the works for a while, so it is great to be able to report on their state as they will soon be making their way to a Pulsar near you! These updates include news on a new "pulsar-updater" package designed to help you keep up to date with our latest releases, a big new feature for one of our core packages for working with Markdown documents and bringing some of our dependencies up to date so you no longer have to keep around old toolchains.
+
+So lets get on with it!
+
+## Update to markdown-preview language identifiers
+
+<img src="./assets/markdown.png" height=200>
+
+[@confused-techie] has been working on a significant update to the way the `markdown-preview` package handles the highlighting of code inside code fences. The `markdown-preview` package is a package included with Pulsar that allows you to render Markdown documents directly within Pulsar without the need for any external tools. One of its features is the fact that it is able to perform syntax highlighting of code within "code fences" or "code blocks". That is, if you include a small snippet of javascript or shell script into your document, `markdown-preview` can provide the same level of syntax highlighting to your snippet in the preview pane as in the main editor.
+
+This is done by providing a "language identifier" to the codeblock in order to signal to Pulsar what language you want it to highlight. For example, if you wanted to embed some javascript in a document, you would need to start the code fence like this:
+
+````
+```javascript
+````
+
+or 
+
+````
+```js
+````
+
+The issue was that this list of identifiers had not been updated for a long time, and people used to other tools and applications with this same functionality (for example, within GitHub and various static site generators) were finding that identifiers they commonly used were not being correctly identified.
+
+This change adds some significant improvements to this system, not content with *only* updating the existing list of languages to match GitHub's [Liguist](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml) libary but massively extending the system to allow users to tailor the behaviour to suit their own needs. In addition to the "Linguist" (GitHub) mode, it adds in lists for other tools such as "Chroma" (Codeberg, Gitea, Hugo), "Rouge" (GitLab, Jekyll) and "HighlightJS" (Markdown-IT). In many cases, these will be identical and users can leave it on the existing default mode (Liguist). However, for those who rely on Pulsar and the `markdown-preview` package to correctly display code as it will be used on the platform of their choice, this allows them to not have to modify the identifier between their editor and their production files.
```

Comment:
```suggestion
This change adds some significant improvements to this system, not content with *only* updating the existing list of languages to match GitHub's [Linguist](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml) library but massively extending the system to allow users to tailor the behaviour to suit their own needs. In addition to the "Linguist" (GitHub) mode, it adds in lists for other tools such as "Chroma" (Codeberg, Gitea, Hugo), "Rouge" (GitLab, Jekyll) and "HighlightJS" (Markdown-IT). In many cases, these will be identical and users can leave it on the existing default mode (Linguist). However, for those who rely on Pulsar and the `markdown-preview` package to correctly display code as it will be used on the platform of their choice, this allows them to not have to modify the identifier between their editor and their production files.
```